lib/ansible/plugins/lookup/credstash.py

```diff
@@ -41,6 +41,8 @@
         description: AWS session token
         env:
           - name: AWS_SESSION_TOKEN
+      default:
+        description: return value if the key is not in the credstash table
```

Comment:
missing version_added

Reply:
Ah ok, what should I say the version will be? `version_added: "2.5"`   maybe?

Reply:
yep

Reply:
Thanks, I've added it as requested.